src/vertical.rs

```diff
@@ -213,7 +213,7 @@ fn rewrite_aligned_items_inner<T: AlignedItem>(
     force_trailing_separator: bool,
 ) -> Option<String> {
     // 1 = ","
-    let item_shape = Shape::indented(offset, context.config).sub_width(1)?;
+    let item_shape = Shape::indented(offset, context.config).sub_width(0)?;
```

Comment:
I don't think this is the correct approach. First because CI is failing, but also because we `.sub_width(1)` to make sure that the field + trailing comma doesn't exceed the max_width. I think you'll need to look a little deeper into this to understand what shape gets used to rewrite the comments.

Reply:
@ytmimi Thanks for the review. Will try to look deeper into this issue!

Reply:
feel free to reach out here or on Zulip if you have any questions

Reply:
Thank you!